django-stubs/contrib/sessions/backends/base.pyi

```diff
@@ -13,17 +13,19 @@ class SessionBase(dict[str, Any]):
     modified: bool
     serializer: Any
     def __init__(self, session_key: str | None = ...) -> None: ...
+    @property
+    def key_salt(self) -> str: ...
     def set_test_cookie(self) -> None: ...
     def test_cookie_worked(self) -> bool: ...
     def delete_test_cookie(self) -> None: ...
     def encode(self, session_dict: dict[str, Any]) -> str: ...
     def decode(self, session_data: bytes | str) -> dict[str, Any]: ...
-    def has_key(self, key: Any) -> Any: ...
-    def keys(self) -> Any: ...
-    def values(self) -> Any: ...
-    def items(self) -> Any: ...
+    def has_key(self, key: Any) -> bool: ...
     def clear(self) -> None: ...
     def is_empty(self) -> bool: ...
+    def _get_new_session_key(self) -> str: ...
```

Comment:
we usually don't type protected methods, why do you need this?

Reply:
`_get_or_create_session_key` may needed to be used when overriding `SessionStore`. For example:

https://github.com/QueraTeam/django-qsessions/blob/ecfd7327ebe32483746704b59fca72f05be07284/qsessions/backends/db.py#LL43C14-L43C14 

I don't have any use cases for the others, typed them just in case. I could remove those.

Reply:
Please, remove them :)
We can always bring them back if there are real use-cases.